### PR TITLE
correct unit test

### PR DESCRIPTION
**Pull request by singharsh0**

### CODE REVIEW

**Comment by singharsh0** [comment]
Hacktoberfest - 22

File: apps/http-sample/src/app.controller.spec.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ describe('AppController', () => {
 
   describe('root', () => {
     it('should return "Hello World!"', () => {
-      expect(appController.getHello()).toBe('Hello World!');
+      const response = appController.getHello();
+      expect(response).toBe('Hello World!');
```
Comment (jmcdo29) [inline apps/http-sample/src/app.controller.spec.ts:20]: Does this really bring any value? [If anything it breaks consistency](https://github.com/jmcdo29/testing-nestjs/blob/main/apps/complex-sample/src/app.controller.spec.ts#L19)

Comment (singharsh0) [reply apps/http-sample/src/app.controller.spec.ts:20]: It is not making any changes as per coverage but this is the coding standard accepted in the industry,
directly putting function in the expect making the code readability not so easy.

Comment (jmcdo29) [reply apps/http-sample/src/app.controller.spec.ts:20]: Can you point to me where that standard exists?

> directly putting function in the expect making the code readability not so easy.

This feels more like an opinion to me. [Even Jest's own docs have method returns in the `expect`](https://jestjs.io/docs/expect#expectvalue)

Comment (jmcdo29) [reply apps/http-sample/src/app.controller.spec.ts:20]: [Two more examples with `expect(function()).assertion())`](https://jestjs.io/docs/expect#tocontainitem)

**Comment by jmcdo29** [comment]
If you can point to where it's industry standard to not inline the method call, I'll re-open this PR and remove the label. After 5 years in the JS world I've not once seen "industry standard" claiming such a thing.

**Pull request closed without merging**